app/models/service_ansible_playbook.rb

```diff
@@ -100,7 +100,12 @@ def get_job_options(action)
   def save_job_options(action, overrides)
     job_options = options.fetch_path(:config_info, action.downcase.to_sym).slice(:hosts, :extra_vars).with_indifferent_access
     job_options[:extra_vars].try(:transform_values!) do |val|
-      val.kind_of?(String) ? val : val[:default] # TODO: support Hash only
+      tmp_val = val.kind_of?(String) ? val : val[:default] # TODO: support Hash only
+      begin
+        JSON.parse(tmp_val)
+      rescue JSON::ParserError
```

Comment:
@fdupont-redhat 
Would it make sense to create a function like mentioned on this post
https://stackoverflow.com/questions/9804777/how-to-test-if-a-string-is-json-or-not

Reply:
@mkanoor 
It makes perfectly sense. See https://github.com/ManageIQ/manageiq/pull/17905/commits/67e5e0e14a3a605c0b9070dd0abc23cf4be2bf3c.